client/tailwind.config.js

```diff
@@ -1,108 +1,21 @@
-/** @type {import("tailwindcss").Config} */
+/** @type {import('tailwindcss').Config} */
 module.exports = {
   content: ["./**/src/**/*.{tsx, ts}"],
   theme: {
     extend: {
+      colors: {
+        gray: "#D9D9D9",
+        pink: "#FF2480",
+        getStarted: "#B7194A",
+        learnMore: "#B7194A",
+        divider: "ADADAD",
+        start: "#B7194A",
+      },
       fontFamily: {
-        sans: ["ui-sans-serif", "Lexend", "system-ui"],
-        display: ["Poppins", "system-ui"],
-        body: ["Lexend", "system-ui"],
+        poppins: "'Poppins', sans-serif",
       },
-      colors: {
-        primary: {
-          100: "#FFFFFF",
-          99: "#FFFBFF",
-          95: "#FFECED",
```

Comment:
This branch probably has merge conflict with the main branch at some point, but the resolving here is bad.
This reverts every change from #18, removing all added colour palette and fonts.